auto_submit/lib/requests/update_revert_reviews.dart

```diff
@@ -0,0 +1,80 @@
+// Copyright 2022 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:auto_submit/exception/bigquery_exception.dart';
+import 'package:auto_submit/model/big_query_revert_request_record.dart';
+import 'package:auto_submit/service/bigquery.dart';
+import 'package:auto_submit/service/github_service.dart';
+import 'package:github/github.dart';
+import 'package:shelf/shelf.dart';
+
+import '../service/log.dart';
+import '../server/authenticated_request_handler.dart';
+
+class UpdateRevertReviews extends AuthenticatedRequestHandler {
+  const UpdateRevertReviews({
+    required super.config,
+    required super.cronAuthProvider,
+  });
+
+  @override
+  Future<Response> get() async {
+    BigqueryService bigqueryService = await config.createBigQueryService();
+
+    log.info('Updating closed revert request review issues in the database.');
+    List<RevertRequestRecord> revertRequestRecords =
+        await bigqueryService.selectOpenReviewRequestIssueRecordsList(projectId: 'flutter-dashboard');
+
+    if (revertRequestRecords.isEmpty) {
+      return Response.ok('No open revert reviews to update.');
+    }
+
+    GithubService githubService = await config.createGithubService(RepositorySlug('flutter', 'flutter'));
+
+    for (RevertRequestRecord revertRequestRecord in revertRequestRecords) {
+      await updateIssue(
+        revertRequestRecord: revertRequestRecord,
+        bigqueryService: bigqueryService,
+        githubService: githubService,
+      );
+    }
+
+    return Response.ok('Finished processing revert review updates.');
+  }
+
+  /// Update the review isse associated with the revert request if it has been
+  /// closed.
+  Future<bool> updateIssue({
+    required RevertRequestRecord revertRequestRecord,
+    required BigqueryService bigqueryService,
+    required GithubService githubService,
+  }) async {
+    bool updated = false;
+    log.info('Processing review issue# ${revertRequestRecord.reviewIssueNumber}.');
+    try {
+      // Get the revert review issue.
+      Issue issue = await githubService.getIssue(
+        slug: RepositorySlug('flutter', 'flutter'),
+        issueNumber: revertRequestRecord.reviewIssueNumber!,
+      );
+
+      if (issue.isClosed) {
```

Comment:
What do we want to do for open issues? Hopefully reviewers can close them in a timely manner. If not, do we want to send a reminder? Not needed to implement in this PR though.

Reply:
Oh that is a good question. I think we could send out periodic update reminders. I'm going to elevate the priority of the review requests to P0 upon feedback from Ian so it will be seen but this is a good idea. Stuff can get lost.

Reply:
I think for the reminder we can just have this cron job send emails for open issues. We can just check against the timestamp and send email reminders on an interval.

Reply:
A github comment should be enough, as it will send emails as well. It also bumps the bug to the weekly critical issue triage.

Reply:
Ah good call. I was looking through the api and it appears that unless you have an email set to public which many do not we would not be able to send emails without significant work involved.

Reply:
I'll add this when I update for the new functionality.